src/main/java/VASSAL/build/module/map/KeyBufferer.java

```diff
@@ -113,24 +114,26 @@ public void mousePressed(MouseEvent e) {
     if (p != null) {
       filter = (EventFilter) p.getProperty(Properties.SELECT_EVENT_FILTER);      
       if (!e.isPopupTrigger() && Boolean.TRUE.equals(p.getProperty(Properties.NON_MOVABLE))) {
```

Comment:
Here is where you want: 

if (!e.isPopupTrigger() && SwingUtils.isLeftMouseButton(e) && Boolean.TRUE.equals(p.getProperty(Properties.NON_MOVABLE))) {